### PR TITLE
almost finish homepage + fix footer

### PR DESCRIPTION
**Pull request by kyulol**
Create a design for the homepage, and finally fix this fucking footer, now i go dodo
PSG EST EN FINALE OMG OMG OMG OMG 3 zero LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOL
@ebblo @Johann-G

### CODE REVIEW

File: app/assets/stylesheets/components/_footer.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+footer{
+  // position: absolute;
+  // bottom: 0;
+  // left: 0;
+}
```
Comment (ebbingloic) [inline app/assets/stylesheets/components/_footer.scss:5]: New line missing.

File: app/assets/stylesheets/pages/_home.scss [hunk 2]
```diff
@@ -1 +1,45 @@
...
+  object-fit: cover;
+  object-position: center;
+}
+.card-home-body{
+  height: 310px;
+}
+
+.tag-category{
+  border-radius: 15px;
+}
```
Comment (ebbingloic) [inline app/assets/stylesheets/pages/_home.scss:45]: New line missing.

File: app/views/pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<div class="container-fluid p-0">
+
+  <%# BANNER %>
```
Comment (ebbingloic) [inline app/views/pages/home.html.erb:3]: Commentaire volontaire ?

File: app/views/pages/home.html.erb [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          <span>35€</span>
+        </div>
+      </div>
+    </div>
+    <div class="card-home col-10 col-md-5 mx-2 p-0 bg-white shadow-sm">
+      <img src="https://cdn.pixabay.com/photo/2015/11/19/10/38/food-1050813_960_720.jpg" class="card-home-img" alt="">
+      <div class="card-home-body border p-4 text-center">
+        <span class="bg-primary py-1 px-3 tag-category text-white">Italian</span>
+        <h5 class="my-3">Our plates selection</h5>
+        <p>Our City Collection features a wide range of carefully citizen who love to cook all around the world - exclusive to HungrYnB. So sad we only have one stomach, we could eat all day long !</p>
```
Comment (ebbingloic) [inline app/views/pages/home.html.erb:50]: 😂

**Review by ebblo — APPROVED** [review]

**Pull request merged**